wapitiCore/main/wapiti.py

```diff
@@ -469,7 +469,12 @@ async def attack(self, stop_event: asyncio.Event):
                         await attack_module.attack(original_request)
 
                     if (datetime.utcnow() - start).total_seconds() > self._max_attack_time >= 1:
-                        logging.info(_("Max attack time was reached for module {0}, stopping.".format(attack_module.name)))
+                        # FIXME: Right now we cannot remove the pylint: disable line because the current I18N system
+                        # uses the string as a token so we cannot use f string
+                        # pylint: disable=consider-using-f-string
+                        logging.info(
```

Comment:
Won't work here as the formating is taking place before the translation but you can do something like:

```python
                        logging.info(
                            _("Max attack time was reached for module %s, stopping."), attack_module.name
                        )
```

It would require to fix the .po files but it is ok.